sdk/nodejs/api/client.gen.ts

```diff
@@ -171,7 +191,16 @@ export type DirectoryEntriesOpts = {
 }
 
 export type DirectoryWithDirectoryOpts = {
+  /**
+   * Exclude artifacts that match the given pattern.
+   * (e.g. ["node_modules/", ".git*"])
+   */
   exclude?: string[]
+
+  /**
+   * Include only artifacts that match the given pattern.
+   * (e.g. ["app/", "package.*"])
```

Comment:
You can also use `@example` here to improve docs readability. (https://typedoc.org/tags/example/)